front/style/pages/_procuration.scss

```diff
@@ -126,6 +126,13 @@
                     min-width: 0;
                 }
             }
+
+            #nbOfProxiesProposed {
+                label:before {
+                    content: none;
```

Comment:
out of curiosity why we need to reset the content?

Reply:
otherwise the global css apply for checkboxes and it looked like this : 

![capture du 2019-02-22 11-27-02](https://user-images.githubusercontent.com/5656168/53236542-cf26d880-3694-11e9-925c-8ba45f62de64.png)

but we worked on it with @thomasbisignani and now it looks better, just updated PR details with new css